### ISSUE TITLE
support for 'include' arrays

### ISSUE THREAD

**Issue by WilliamSalusky** [x0rz/phishing_catcher#9]
Create support to read custom include configs for suspicious_keywords, highly_suspicious and suspicious_tld arrays so that every commit avoids stomping on local changes.

**Comment by Ge0** [comment]
You mean putting those arrays in file configurations?

**Comment by WilliamSalusky** [comment]
Yes. That was a better way to phrase it!  Place configurable array's in an external config file from the primary .py
